connectors/connector-greenhouse/server.ts

```diff
@@ -1,8 +1,28 @@
+import {initGreenhouseSDK, type greenhouseTypes} from '@opensdks/sdk-greenhouse'
 import type {ConnectorServer} from '@openint/cdk'
 import type {greenhouseSchema} from './def'
 
-export const greenhouseServer = {} satisfies ConnectorServer<
-  typeof greenhouseSchema
+export type GreenhouseSDK = ReturnType<typeof initGreenhouseSDK>
+
+export type GreenhouseTypes = greenhouseTypes
+
+export type GreenhouseObjectType = GreenhouseTypes['components']['schemas']
+
+export const greenhouseServer = {
+  newInstance: ({config}) => {
```

Comment:
Were you able to test this out? Did this work? (I haven't gotten a chance to really test after the refactor yet.